nwbwidgets/base.py

```diff
@@ -213,51 +212,52 @@ def plot_traces(time_series: TimeSeries, time_start=0, time_duration=None, trace
         Index range of traces to view
     title: str, optional
     ylabel: str, optional
-
     Returns
     -------
-
     """
-
+    
+    if type(time_series.data) != np.ndarray:
+        time_series.data = np.asarray(time_series.data)
+    
     if time_start == 0:
         t_ind_start = 0
     else:
         t_ind_start = timeseries_time_to_ind(time_series, time_start)
+    
     if time_duration is None:
         t_ind_stop = None
     else:
         t_ind_stop = timeseries_time_to_ind(time_series, time_start + time_duration)
-
-    if trace_window is None:
-        trace_window = [0, time_series.data.shape[1]]
-    tt = get_timeseries_tt(time_series, t_ind_start, t_ind_stop)
-    if time_series.data.shape[1] == len(tt):  # fix of orientation is incorrect
```

Comment:
This line is meant to check if the data was accidentally stored with time in the second dimension and switch the data if that is the case.

Reply:
Correct.